### PR TITLE
[feat] added adoptee address info to supabase

### LINKED ISSUE #189: pull adoptee mailing address info
yay great job last week!! 🌸

This week, you’ll be pulling altering our pipeline to populate the `facility` col in our adoptee vector database and creating a new table in supabase with 4 cols: facility id, facility name, system, and mailing address. Once this is ready, we will be able to display the mailing info for a given adoptee on “active” status application cards. There’s 3 main parts to this:

**1.  alter our pipeline to pull adoptee facility information, should be the “Facility” col of the WL PIPs board** 
  - you should pull the monday id for the given facility item/row
  - run the pipeline manually (should be simple once you understand `api/set_embeddings/pipeline.py`) once to populate the current vector db. make sure you comment out the vectorization portion while doing this!
  - our existing pipeline is a good example, but you just need the portion that is fetching data. and we won't be running it on a cron job or vectorizing anything so it's greatly simplified

**2.  create a pipeline to pull the facility table info from the “Facilities” board (ID ends in 4455) and populate a new supabase table called `adoptee_facilities` with 4 cols (and 1.3k or so rows)**	
- `facility_id` (text): should be the monday id for the row, and will match with what populates the `facility` col in `adoptee_vector_test`
- `facility_name` (text): should be the name of the facility, shows up as the first col on the monday board,  
- `system` (text): corresponds to the “System” col on the board (i.e. has color labels for State, County, etc)
- `mailing_address` (text): corresponds to “Inmate Mailing Address (Letters)” 
- **note, pull from all the groups, except for “Not in Custody”

**3. display on the app cards in `ApplicationPreviewDialog.tsx`**
- have one display field for the facility name and one for the address
- you’ll have to alter `applications/getApplicationWithAdoptees.ts` and potentially other parts of [jinkang’s PR](https://github.com/calblueprint/adopt-an-inmate/pull/174) to ensure `appData` has what you need for displaying
- jinkang is currently working on a hi-fi version of the app card pop-ups/modals so if it displays unstyled in your PR that’s fine!

This is quite the involved sprint, so let me know if you need help or have questions.

### PR DESCRIPTION
**Pull request by saitunguturu**
[//]: # "Feel free to customize this template and the emojis to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
PR adds facility info to the adoptee pipeline and creates a new facilities table in Supabase

- Added facility_id column to the adoptee pipeline to pull the linked facility item ID from the "Facility" connect boards column on the WL PIPs board
- Created _fetch_facilities.py to fetch all facilities from the Facilities board (excluding "Not in Custody") and populates the new adoptee_facilities Supabase table with facility_id, facility_name, system, and mailing_address (one-time use only, so it's not really needed now)
- Created _update_facility.py, a one-time script that links each adoptee in adoptee_vector_test to their corresponding facility via facility_id
- Both scripts were run manually to fill in the tables

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

can't really screenshot since it's open source, but the changes affected adoptee_vector_test and adoptee_facilities

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
I am not sure how to test the pipeline other than adding new adoptees

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @InayaY3

### CODE REVIEW

File: api/set_embeddings/_fetch_facilities.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+)
+
+load_dotenv(os.path.join(os.path.dirname(__file__), "../../.env.local"))
+
+SUPABASE_URL = os.getenv("NEXT_PUBLIC_SUPABASE_URL")
+SUPABASE_KEY = os.getenv("SUPABASE_SERVICE_KEY")
+supabase = create_client(SUPABASE_URL, SUPABASE_KEY)
+
+FACILITIES_BOARD_ID = 6465684455
+NOT_IN_CUSTODY_GROUP_ID = "6620891062"
```
Comment (InayaY3) [inline api/set_embeddings/_fetch_facilities.py:19]: generally speaking, we would prob make these .env variables to follow conventions from our other monday API usage but not a big deal. 

board IDs are not super sensitive on their own and this is something we run manually anyway (and don't access anywhere else in the app)

File: api/set_embeddings/_update_facility.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    while True:
+        cursor_str = f', cursor: "{cursor}"' if cursor else ''
+        query = f"""query {{
+          boards(ids: {MONDAY_BOARD_ID}) {{
+            items_page(limit: 500{cursor_str}) {{
+              cursor
+              items {{
+                id
+                group {{ id }}
+                linked_items(link_to_item_column_id: "facility__1", linked_board_id: 6465684455) {{
```
Comment (InayaY3) [inline api/set_embeddings/_update_facility.py:34]: same thing as above about the board ID. just a nit for future reference but not worth changing rn

**Review by InayaY3 — APPROVED** [review]
awesome, lgtm! changes already up on supabase. will inform the NPO about missing records
